src/app/components/user-job-card/user-job-card.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.02);
+  }
+
+  .home-job-item-name-container {
+    height: 110px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
```

Comment:
Expected item on line 29 to appear before line 27. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#declarationorder)